content/en/blog/2019/announcing-1.3/index.md

```diff
@@ -0,0 +1,63 @@
+
+- Improve the experience of new users adopting Istio
+- Improve the experience of users debugging problems
+- Support more applications without any additional configuration
+
+Every few releases, the Istio team gets excited as we make dramatic improvements to usability, APIs, and the overall system performance. This release is one of those where the team is excited to roll out some key updates.
+
+## Intelligent protocol detection
+
+In past releases, you had to explicitly declare the protocol for service ports by using a special port naming format. This requirement caused problems for users that did not name their ports when they added their applications to the mesh. Starting with 1.3, the protocol on the wire will be automatically detected (as HTTP or TCP), eliminating the need to name ports explicitly. We will be adding support for additional [non-HTTP] protocols in future releases.
+
+## Mixer-less telemetry (experimental)
+
+Yes you read that right! We had already turned off the `istio-policy` service by default as we implemented most of the common security policies such as RBAC directly in Envoy. We are now on track to migrate most of the mixer's telemetry functionality as well into Envoy. As of this release, if all you care about is telemetry for HTTP services, we have enhanced the Istio proxy to emit HTTP metrics directly to Prometheus, without requiring the `istio-telemetry` service to enrich the information. You can experiment with the mixer-less telemetry feature for HTTP services by following these [instructions](https://github.com/istio/istio/wiki/Mixerless-HTTP-Telemetry). We will be polishing this feature in the coming months including adding telemetry support for TCP services when Istio mutual TLS is being used.
+
+## Container ports are no longer required
+
+In the past, Istio required that pods explicitly declare the Kubernetes `containerPort` for each container as a security measure against trampolining traffic. Istio 1.3 has a secure and simpler way of handling all inbound traffic on any port into a Pod without requiring the `containerPort` declarations. We have also completely eliminated infinite loops caused in the IPtables rules when pods send traffic to themselves.
+
+## Customize generated Envoy configuration
+
+While Istio 1.3 focuses on out of box usability, it also caters to the expert users who would like to use advanced features in Envoy that Istio does not expose via its Networking APIs. We have enhanced the `EnvoyFilter` API to allow users to fully customize the Envoy HTTP route configuration returned by RDS, HTTP/TCP listeners and their filter chains returned by LDS, and the set of clusters returned by CDS. You get the best of both worlds - leverage  Istio to integrate with Kubernetes, handle large fleets of Envoys in an efficient manner and still retain the ability to customize the generated Envoy configuration to meet specific requirements within your infrastructure.
```

Comment:
Is this also experimental?  I don't see much doc or testing around this.

Reply:
These are API improvements.  Do we have to prefix everything in the thing with experimental? in that case, we might as well frame that sentence as "the following are the experimental features".. but then saying protocol sniffing is experimental and then stating that it gets enabled by default is not nice.